mne_bids/copyfiles.py

```diff
@@ -153,6 +154,69 @@ def copyfile_ctf(src, dest):
                   op.join(dest, bids_folder_name + ext))
 
 
+def copyfile_kit(src, dest, subject_id, session_id,
+                 task, run, _init_kwargs):
+    """Copy and rename KIT files to a new location.
+
+    Parameters
+    ----------
+    src : str
+        Path to the source raw .con or .sqd folder.
+    dest : str
+        Path to the destination of the new bids folder.
+    subject_id : str | None
+        The subject ID. Corresponds to "sub".
+    session_id : str | None
+        The session identifier. Corresponds to "ses".
+    task : str | None
+        The task identifier. Corresponds to "task".
+    run : int | None
+        The run number. Corresponds to "run".
+    _init_kwargs : dict
+        Extract information of marker and headpoints
+
+    """
+    from mne_bids.write import make_bids_basename
+    # KIT data requires the marker file to be copied over too
+    sh.copyfile(src, dest)
+    data_path = op.split(dest)[0]
+    if 'mrk' in _init_kwargs:
+        hpi = _init_kwargs['mrk']
+        acq_map = dict()
+        if isinstance(hpi, list):
+            if _get_mrk_meas_date(hpi[0]) > _get_mrk_meas_date(hpi[1]):
+                raise ValueError('Markers provided in incorrect order.')
+            _, marker_ext = _parse_ext(hpi[0])
+            acq_map = dict(zip(['pre', 'post'], hpi))
+        else:
+            _, marker_ext = _parse_ext(hpi)
+            acq_map[None] = hpi
+        for key, value in acq_map.items():
+            marker_fname = make_bids_basename(
+                subject=subject_id, session=session_id, task=task, run=run,
+                acquisition=key, suffix='markers%s' % marker_ext,
+                prefix=data_path)
+            sh.copyfile(value, marker_fname)
+    if 'elp' in _init_kwargs:
+        hpi_file = _init_kwargs['elp']
+        task, run, acp = None, None, 'ELP'
+        marker_ext = '.pos'
+        marker_fname = make_bids_basename(
+            subject=subject_id, session=session_id, task=task, run=run,
+            acquisition=acp, suffix='headshape%s' % marker_ext,
```

Comment:
actually thinking a bit more about it, why do you overwrite the `acq` that the user provides? It makes sense in the case of 'mrk' but not sure if it makes sense here.

Reply:
i tried several combinaiton for a validated BIDS spec
only `sub-108/meg/sub-108_acq-ELP_headshape.pos` works.

otherwise:
`sub-108/meg/sub-108_acq-ELP_headshape.elp`
`sub-108/meg/sub-108_elp.pos` etc failed

i actually am not sure which one is a better name though